src/test/java/com/ibm/watson/developer_cloud/service/GenericServiceTest.java

```diff
@@ -62,6 +87,45 @@ private RecordedRequest checkRequest() throws InterruptedException {
   }
 
   /**
+   * Checks the credential constructors of all services.
+   */
+  @Test
+  public void testConstructors() {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://www.codacy.com/app/watson-developer-cloud/java-sdk/file/2724346036/issues/source?bid=3303699&fileBranchId=3347904#l93)